mmv1/templates/terraform/examples/privateca_certificate_config.tf.erb

```diff
@@ -1,62 +1,79 @@
 resource "google_privateca_certificate_authority" "test-ca" {
-  provider = google-beta
   certificate_authority_id = "<%= ctx[:vars]["certificate_authority_id"] %>"
   location = "us-central1"
+  pool = "<%= ctx[:vars]["pool"] %>"
   tier = "ENTERPRISE"
+  ignore_active_certificates = true
```

Comment:
this flag is only relevant on the DeleteCertificateAuthority request message (it's essentially a kind of "force-delete" flag). It feels weird to add "ignore active certificates" when defining a CA.

Is there a normal way that such flags are dealt with? If it needs to be part of the CA's definition, then perhaps a better name could be used, e.g. "ignore_active_certificates_on_deletion". WDYT?